app/src/main/java/com/nextcloud/talk/activities/MainActivity.kt

```diff
@@ -229,7 +229,7 @@ class MainActivity : BaseActivity(), ActionBarProvider {
         val roomType = "1"
         val currentUser = userUtils.currentUser ?: return
 
-        val apiVersion = ApiUtils.getConversationApiVersion(currentUser, intArrayOf(1))
+        val apiVersion = ApiUtils.getConversationApiVersion(currentUser, intArrayOf(ApiUtils.APIv4, 1))
```

Comment:
for consistency probably also add `ApiUtils.APIv1` :)

Reply:
let me do that once done, so my other PRs and branches don't break on the rebases even more :D